lib/linguist/strategy/manpage.rb

```diff
@@ -0,0 +1,42 @@
+module Linguist
+  module Strategy
+    # Detects man pages based on numeric file extensions with group suffixes.
+    class Manpage
+
+      # Public: RegExp for matching conventional manpage extensions
+      #
+      # This is the same expression as that used by `github/markup`
+      MANPAGE_EXTS = /\.(?:[1-9](?![0-9])[a-z_0-9]*|0p|n|man|mdoc)(?:\.in)?$/i
```

Comment:
That regular expression is also going to match the following extensions from other languages:
- Forth: `.4th`.
- Perl6: `.6pl` and `.6pm`.
- TI Program: `.8xp` and `.8xk`.

Reply:
Will it? `Linguist::Strategy::Manpage` comes after `Linguist::Strategy::Extension`, which is deliberately to address the problem you pointed out.

Reply:
Ah, right. The tests would also fail if someone changed the order of strategies, so :+1: for me!